src/ui/CameraShareGuide.tsx

```diff
@@ -1,11 +1,43 @@
 import React, { Suspense } from 'react';
 import Markdown from 'react-markdown';
+import rehypeSlug from 'rehype-slug';
+import rehypeRaw from 'rehype-raw';
```

Comment:
Check the  package size of this and add the details of this in the description. If it's increasing the bundle size then it will affect the proctoring library package and it's performance. We should be looking for lightweight packages and if possible not use it if its too big.